frontend/packages/design-system/src/tokens/typography.ts

```diff
@@ -30,30 +6,49 @@ export default {
         italic: 'Roboto_400Regular_Italic',
       },
       500: {
-        medium: 'Roboto_500Medium',
+        normal: 'Roboto_500Medium',
       },
       700: {
         normal: 'Roboto_700Bold',
       },
     },
   },
+  fontWeights: {
+    regular: 400,
+    medium: 500,
+    bold: 700,
+  },
   fonts: {
-    display: 'Roboto',
-    heading: 'Roboto',
-    title: 'Roboto',
-    bodyText: 'Roboto',
-    inline: 'Roboto',
-    date: 'Roboto',
-    label: 'Roboto',
-    caption: 'Roboto',
+    body: 'Roboto',
+    mono: undefined,
+  },
+  fontSizes: {
+    '3xs': 14,
```

Comment:
do these values adjust if the base font size changes? or did you give up on that?

Reply:
Gave up on it. I think it's better to just keep it in sync with design for now and come back to that later if it makes sense